machine.go

```diff
@@ -263,6 +267,11 @@ func (m *Machine) Shutdown(ctx context.Context) error {
 	return m.sendCtrlAltDel(ctx)
 }
 
+// ExitC returns machine's exit channel
+func (m *Machine) ExitCh() <-chan struct{} {
```

Comment:
If you expose m.exitCh here, it can be drained outside the SDK and break `Wait`.  The channel semantics (delivery once) isn't really what we want; instead, we want a mechanism where callers can subscribe to notifications and they can be delivered without breaking the SDK's use of exitCh.

Reply:
SDK closes `exitCh` when VM exits instead of writing to it, so there is no drain.
We can have as many readers as we need, they all will be blocked until `close` call.

Reply:
Added a comment to reflect this.